### PR TITLE
Fix / Extend missing translations

### PR DESCRIPTION
**Pull request by jpirnay**
Fix / Extend missing translations
a) DE-Locale
b) Fix in Simulation-Window
c) Add missing tooltip

### CODE REVIEW

File: translate.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import os, polib
```
Comment (Sophist-UK) [inline translate.py:1]: Please split this line into two separate imports.

File: translate.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+				doit = 0 
+			if doit == 1:
+				print("Translate " + d + po_file)
+				po = polib.pofile(d + po_file)
+				po.save_as_mofile(d + mo_file)
+				mo_files.append(d + mo_file)
+		data_files.append((d, mo_files))
+	return data_files
+	
+data_files=create_mo_files()
```
Comment (Sophist-UK) [inline translate.py:31]: I think this should just be `create_mo_files()`.

File: translate.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+	for l in next(os.walk(localedir))[1]]
+	for d in po_dirs:
+		mo_files = []
+		po_files = [f
+					for f in next(os.walk(d))[2]
+					if os.path.splitext(f)[1] == '.po']
+		for po_file in po_files:
+			filename, extension = os.path.splitext(po_file)
+			mo_file = filename + '.mo'
+			doit = 1
```
Comment (Sophist-UK) [inline translate.py:16]: `doit` should use `True` and `False` rather than 1 and 0.

**Review by Sophist-UK — CHANGES_REQUESTED** [review]

**Review by tatarize — APPROVED** [review]
The translate.py thing is fine, it's mostly a non-integrated standalone script and doesn't really need to exist, but it's fine if it does. polib isn't really a requirement here and, in theory, it would fall to the wayside if `poboy` got finished up to a nice and workable state.

I think it's fine to just hit the script with black and isort and merge it in normally.

File: translate.py (hunk 2 above)
Comment (jpirnay) [reply translate.py:31]: Will amend

File: translate.py (hunk 3 above)
Comment (jpirnay) [reply translate.py:16]: Right...

File: translate.py [hunk 4]
```diff
@@ -0,0 +1,31 @@
+import os, polib
+
+def create_mo_files():
+	data_files = []
+	localedir = './locale'
+	po_dirs = [localedir + '/' + l + '/LC_MESSAGES/'
+	for l in next(os.walk(localedir))[1]]
```
[the remark this answers is not in the document]
Comment (jpirnay) [reply translate.py:7]: I've 'black'ed the script...

**Comment by Sophist-UK** [comment]
Closing in favour of #812

**Pull request closed without merging**